pep-0458.txt

```diff
@@ -369,6 +369,12 @@ generate cryptographic keys, and manage a TUF repository.
 
 __ https://github.com/theupdateframework/tuf/blob/v0.11.1/docs/TUTORIAL.md#how-to-create-and-modify-a-tuf-repository
 
+In standard operation the *bin-n* metadata will be created and signed as new
```

Comment:
```suggestion
In standard operation, the *bin-n* metadata will be updated and signed as new
```

Reply:
Addressed in rebased & force-pushed patches.